Gems/ScriptCanvas/Code/Include/ScriptCanvas/Internal/Nodes/ExpressionNodeBase.cpp

```diff
@@ -267,6 +267,17 @@ namespace ScriptCanvas
                 return true;
             }
 
+            void ExpressionNodeBase::FixDynamicGroupDataType(const SlotId& slotId, Data::Type displayType)
```

Comment:
As per Luis' previous comment, this should be named for what it does in a process.. Like UpdateGroupTypePostChange or something that tells us why it is needed, rather than "Fix".

Reply:
sorry, forgot to fix this